start/resources/jupyter.md

```diff
@@ -0,0 +1,138 @@
+
+
+# Objective
+
+This guide describes how to request an account to access JupyterLab, launch a JupyterLab instance, and how to send jobs from JupyterLab to an execution point dedicated to your JupyterLab instance. 
+
+
+# What is JupyterLab?
+
+JupyterLab is a web-based interface for [Project Jupyter](https://jupyter.org). It is an interactive environment that supports a variety of computational activities and workflows through features such as notebooks, terminals, text editors, etc.
+
+More information about the JupyterLab interface can be found in the [JupyterLab manual](https://jupyterlab.readthedocs.io/en/stable/getting_started/overview.html). 
+
+JupyterLab can be a helpful resource for teaching and for running small batches of HTCondor jobs for testing and/or troubleshooting purposes.  
+
+
+# Steps to Accessing and Working in JupyterLab
+
+The steps necessary for working in JupyterLab are: 
+
+- Request an account.
+- Meet with a Research Computing Facilitator for a short introduction to working with JupyterLab and to activate your account.
+- Launch a JupyterLab instance.
+
+
+# Request Access to a JupyterLab Access Point
+
+## Request an Account 
+
+To request access to JupyterLab, submit an application using the following steps:
+
+(1) Go to the [registration website](https://registry.cilogon.org/registry/co_petitions/start/coef:261). You will be redirected to the CILogon sign in page. Select your institution and use your institutional credentials to login. 
+
+If you have issues signing in using your institutional credentials, contact us at support@opensciencegrid.org.
+
+(2) Once you sign in, you will be redirected to the "JupyterLab User Enrollment for New Users" page. Click "Begin" and enter your name and email address on the following page. In many cases, this information will be automatically populated. If desired, it is possible to manually edit any information automatically filled in. Once you have entered your information, click "SUBMIT". 
+
+(3) After submitting your application, you will receive an email from registry@cilogon.org to verify your email address. Click the link listed in the email to be redirected to a page confirm your invitation details. Click the "ACCEPT" button to complete this step. 
+
+
+## Meet with a Research Computing Facilitator
+
+Once OSG staff receive your email verification, a Research Computing Facilitator will contact you within one business day to arrange a short consultation and introduction to JupyterLab resources. During this meeting, our staff will provide personalized start-up guidance per your specific computational research goals and activate your account.
+You will be notified when your account is approved via email.
+
+Additionally, you will be assigned a "ProjectName" that you can use later in an HTCondor submit file to submit jobs through your JupyterLab interface to HTCondor. 
```

Comment:
```suggestion
Once OSG staff receive your email verification, a Research Computing Facilitator will contact you within one business day to arrange a short consultation and introduction to the JupterLab resources. During this meeting, our staff will provide personalized start-up guidance per your specific computational research goals and activate your account.

Following the meeting, the Facilitator will approve your account, add your profile to any relevant JupyterLab ‘project’ names, and ensure that you have access to the JupyterLab. (You will receive automated emails for some of these actions, which you can otherwise ignore.)
```